### ISSUE TITLE
Installation instructions

### ISSUE THREAD

**Issue by neopostmodern** [grrrr/py#4]
For beginners (e.g. me) it is very hard to follow the installation instructions in the README. It does not state what needs to be downloaded from where, nor where it should be saved. Might be obvious to regular pd users I guess.

(I still haven't figured this out, hints appreciated.)

**Comment by grrrr** [comment]
Hi, i appreciate your comment. However, this is the developer site for py/pyext, as is the README. It is not the user site. If you just want to use py/pyext, you should consult http://grrrr.org/research/software/py . I hope that's clearer.

**Comment by neopostmodern** [comment]
Well, I came to this repo from the website, but the instructions there weren't clear enough for a beginner. I've since figured it out, but maybe you could rephrase the following sentence (or the entire "Downloads" paragraph):

> Once downloaded to your hard drive, the location of these scripts need to be registered in the Pure Data or Max path settings to be found by the py/pyext objects.

Into something like:

> Copy the binaries into the `extras` directory of your installation (e.g. `/usr/lib/pd-l2ork/extras`) and add it to Pd by adding a `py` entry at `File > Startup...`. Next, add the folder with the Python scripts (where you can also put your own) to Pd's path (`File > Path...`).

Maybe there are generic instructions for adding extras somewhere, but I couldn't really find any. If there are, you could link them too.

Would that be possible?

**Comment by bgenchel** [comment]
I agree with OP. The installation instructions on the website are almost even less clear. There are like 4 or 5 different versions of py in your beta folder and I have no idea which one to pick. Additionally, you should update the README here even for developers, because the py.mxo file for Mac doesn't even exist inside this repo.

**Comment by serlecu** [comment]
I agree. In general, makes sense to download the news upload.
For Max users, I downloaded "py.package-i386-local-2.7.zip" and Max recognized the objects on 10.14. with latest. Just copy the py folder to your Externals. Don't forget to include "py-objectmapping.txt" at "/Applications/Max.app/Contents/Resources/C74/init".
